suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml

```diff
@@ -78,7 +78,7 @@ tests:
       config:
         command: add
         id: client.1                      # client Id (<type>.<Id>)
-        node: node8                       # client node
+        node: node7                       # client node
         install_packages:
           - ceph-common
           - ceph-base
```

Comment:
ceph-base is not required for client node. Can you remove the ceph-base.

Reply:
Will remove the unnecessary packages when i work on fixing the suite issues. for now, I have just changed the node for client deployment.

Reply:
Sure Pawan.